doc/psidk/theme/js/mermaid-load.js

```diff
@@ -0,0 +1,13 @@
+const script = document.createElement('script');
+script.src = 'https://cdn.jsdelivr.net/npm/mermaid@10/dist/mermaid.min.js';
```

Comment:
I know I'm in a minority, but I like being able to view documentation offline...

Reply:
You can, your penalty is just that you will have to deal with a text representation of the mermaid diagram 😜